packages/ra-ui-materialui/src/form/TabbedForm.spec.js

```diff
@@ -9,28 +9,11 @@ const translate = label => label;
 const muiTheme = { textField: { errorColor: 'red' } };
 
 describe('<TabbedForm />', () => {
-    it('should render tabs', () => {
```

Comment:
why did you remove this one?

Reply:
Because of React Router, it seems that it doesn't render in tests